doc/releases/changelog-dev.md

```diff
@@ -37,34 +91,42 @@ Users can specify the control wires as well as the values to control the operati
     [Exp(1.570795j, 'PauliY', wires=[0])]
   ```
 
-  Moreover, the obtained tapered operation can be directly used within a QNode:
+  Moreover, the obtained tapered operation can be used directly within a QNode.
 
   ```pycon
     >>> dev = qml.device('default.qubit', wires=[0, 1])
     >>> @qml.qnode(dev)
     ... def circuit(params):
     ...     tap_op(params[0])
-    ...     return qml.expval(qml.PauliZ(0)@qml.PauliZ(1))
+    ...     return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
     >>> drawer = qml.draw(circuit, show_all_wires=True)
     >>> print(drawer(params=[3.14159]))
         0: ─Exp(1.570795j PauliY)─┤ ╭<Z@Z>
         1: ───────────────────────┤ ╰<Z@Z>
-
   ```
 
-<h4>Pauli Module</h4>
+<h4>(TODO: title) Pauli Module</h4>
 
-* Re-organized and grouped all functions in PennyLane responsible for manipulation of Pauli operators into a `pauli` 
+* Reorganized and grouped all functions in PennyLane responsible for manipulation of Pauli operators into a `pauli` 
   module. Deprecated the `grouping` module and moved logic from `pennylane/grouping` to `pennylane/pauli/grouping`.
   [(#3179)](https://github.com/PennyLaneAI/pennylane/pull/3179)
 
+<h4>(TODO: title) New operators and optimizers</h4>
+
+* Optimizing quantum circuits can now be done *adaptively* with `qml.AdaptiveOptimizer`.
+  [(#3192)](https://github.com/PennyLaneAI/pennylane/pull/3192)
+
+  The `qml.AdaptiveOptimizer` optimizer takes an initial circuit and a collection of operators
+  as input and adds a selected gate to the circuits at each optimization step. The process of
+  growing the circuit can be repeated until the circuit gradients converge to zero within a given
+  threshold. The adaptive optimizer can be used to implement algorithms such as `ADAPT-VQE`.
+
+  For a detailed breakdown of its implementation, check out [our demo](https://pennylane.ai/qml/demos/tutorial_adaptive_circuits.html)!
```

Comment:
@trbromley, This is great idea but I think we might have to address it next week since I don't want to just copy over code from the demo (it might be too lengthy).

Reply:
There was code there before but the example was veeeeery long.